src/communication/network_interface_cl.c

```diff
@@ -11602,3 +11599,56 @@ tdes_reset_query_start_info (PT_NODE * node)
     }
 #endif
 }
+
+/* manage_lob_dir - create or delete lob dir
+ *
+ * return:
+ *
+ *   hfid(in): separate lob dir by table
+ *   attrid_arr(in): separate lob dir by column
+ *   lob_arr_length(in): number of LOB-type columns
+ *   mode(in): separate create or delete mode
+ *
+ * NOTE:
+ * manage_lob_dir - request to server to create or delete lob dir
+ */
+int
+manage_lob_dir (HFID * hfid, int *attrid_arr, int lob_arr_length, LOB_DIR_MANAGE_MODE mode)
```

Comment:
구현 코드 전반에 대해 prefix와 naming 규칙 통일을 통해 가독성 향상 필요해 보입니다.
```suggestion
manage_lob_dir (HFID * hfid, int *attrid_arr, int attrid_arr_length, LOB_DIR_MANAGE_MODE mode)
```

Reply:
수정했습니다.